### ISSUE TITLE
De nieuwe WCAG check (2.2) geeft een fout op het contrast van de linker banner 'werkversie'

### ISSUE THREAD

**Issue by NielsHoffmann** [Geonovum/NL-ReSpec-template#18]
Afkomstig uit de WCAG 2.2 check in Raamwerk Geostandaarden:
```
[
  {
    "url": "http://localhost:8080/snapshot.html",
    "id": "color-contrast",
    "impact": "serious",
    "description": "Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds",
    "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=webdriverjs",
    "nodes": [
      {
        "target": [
          ".sidelabel"
        ],
        "html": "<div class=\"sidelabel\">Geonovum Handreiking - Werkversie</div>",
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.99 (foreground color: #ffffff, background color: #ff0000, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
      }
    ]
  }
]
```
Dus ik denk dat er ergens in het template/css iets aangepast moet worden om deze fout op te lossen...
